### ISSUE TITLE
Branch: Dev -- fix the CSS on the experimental homepage

### ISSUE THREAD

**Issue by bravegrumpy** [bravegrumpy/bravegrumpy.website#55]
# Experimental Homepage Styling
The new homepage has created a start for using some templating.

This is a first step towards refactoring some of this into DRY. Like for the navbar, header and footer. 

[x] Use javascript document.write() to add html into the designated location.
[ ] figure out how to use a DOM to have the javascript file actually look at the "included" html file.
[ ] figure out what classes are needed to allow the styling to be applied appropriately. 

Basically, I think I'm re-inventing components.
